packaging_automation/common_tool_methods.py

```diff
@@ -85,6 +86,18 @@ def get_minor_project_version(project_version: str) -> str:
     return f'{project_version_details["major"]}.{project_version_details["minor"]}'
 
 
+def does_project_version_contain_release_stage(project_version: str) -> bool:
```

Comment:
maybe:

```suggestion
def project_version_contains_release_type(project_version: str) -> bool:
```

Reply:
Fixed as you suggested. Only used release_stage instead of release_type